src/main/java/seedu/address/model/project/CompletableTaskList.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.model.project;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CompletableTaskList {
+
+    private final List<Completable> completableTasks = new new ArrayList<>();
```

Comment:
Do add the types in the array list as well. It's failing the checks.
"new" has been duplicated twice.

Reply:
Waiting for your PR @vevek, which would have implemented the required types.